Lib/selectors.py

```diff
@@ -338,92 +338,90 @@ def select(self, timeout=None):
         return ready
 
 
-if hasattr(select, 'poll'):
+class _PollLikeSelector(_BaseSelectorImpl):
+    """Base class shared between poll, epoll and devpoll selectors."""
 
-    class PollSelector(_BaseSelectorImpl):
-        """Poll-based selector."""
+    def __init__(self):
+        super().__init__()
+        self._poller = self._poller()
```

Comment:
Yes, the idea is to have a `_poller` attribute which can be referenced by all 3 classes. Actually I think it'd be cool to do the same also for select and kqueue.
And yes, it's probably better to also have `_poller_cls`.
I don't have a strong opinion about abstract properties but it's up to you.